R/derive_var_extreme_flag.R

```diff
@@ -206,18 +201,29 @@
 #'     new_var = AOCCIFL,
 #'     by_vars = exprs(STUDYID, USUBJID),
 #'     order = exprs(TEMP_AESEVN, AESTDY, AESEQ),
-#'     mode = "first"
+#'     mode = "first",
+#'     check_type = "warning"
 #'   ) %>%
 #'   arrange(STUDYID, USUBJID, AESTDY, AESEQ) %>%
 #'   select(STUDYID, USUBJID, AEDECOD, AESEV, AESTDY, AESEQ, AOCCIFL)
 #'
+#' @info Note here that the presence of `AESEQ` as a sorting variable inside
+#'   the `order` argument ensures that the combination of `by_vars` and
+#'   `order` indexes unique records in the dataset. If this had been omitted,
+#'   the choice of `check_type = "warning"` would have ensured that
+#'   `derive_var_extreme_event()` would throw a warning due to perceived
+#'   duplicate records (in this case, the first two AEs for subject `"1015"`).
+#'   If no sorting variables exist, or if these duplicates are known, then
```

Comment:
```suggestion
#'   If no sorting variables exist, or if these duplicates are acceptable, then
```
Does this wording work better?